### PR TITLE
done with patient details and loading page ui 

### PR DESCRIPTION
**Pull request by Abel-Mekonen**
check out the changes

### CODE REVIEW

**Review by tamiratdereje — APPROVED** [review]

File: afyacare/lib/presentation/screens/pateint_details.dart [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+class _PatientDetailsState extends State<PatientDetails> {
+  final _recordFormKey = GlobalKey<FormState>();
+  TextEditingController recordDescription = TextEditingController();
+  TextEditingController prescribeMed = TextEditingController();
+  List<String> chipsList = chips().chipsList;
+
+  Widget createChip(String name) {
+    return Padding(
+      padding: const EdgeInsets.all(6),
```
Comment (jebessaD) [inline afyacare/lib/presentation/screens/pateint_details.dart:28]: change the chip to a pill shape

File: afyacare/lib/presentation/screens/pateint_details.dart [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+  State<PatientDetails> createState() => _PatientDetailsState();
+}
+
+class _PatientDetailsState extends State<PatientDetails> {
+  final _recordFormKey = GlobalKey<FormState>();
+  TextEditingController recordDescription = TextEditingController();
+  TextEditingController prescribeMed = TextEditingController();
+  List<String> chipsList = chips().chipsList;
+
+  Widget createChip(String name) {
```
Comment (jebessaD) [inline afyacare/lib/presentation/screens/pateint_details.dart:26]: add onDelete icon

File: afyacare/lib/presentation/screens/pateint_details.dart [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        children: [
+          const circleClip(),
+          Padding(
+            padding: const EdgeInsets.all(20.0),
+            child: Stack(
+              children: [
```
Comment (jebessaD) [inline afyacare/lib/presentation/screens/pateint_details.dart:58]: change main axis alignment to start

File: afyacare/lib/presentation/screens/pateint_details.dart [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+                        Text(
+                          "Pateint",
+                          style: AfyaTheme.lightTextTheme.headline2,
+                        ),
+                        Text(
+                          "Details",
+                          style: AfyaTheme.lightTextTheme.headline2,
+                        ),
+                        const SizedBox(
+                          height: 70,
```
Comment (jebessaD) [inline afyacare/lib/presentation/screens/pateint_details.dart:76]: decrease sizedbox height

**Review by jebessaD — COMMENTED** [review]
patient detail page/ui reviewed

File: afyacare/lib/main.dart [hunk 5]
```diff
@@ -14,13 +15,11 @@ class MyApp extends StatelessWidget {
...
     return MaterialApp(
-      debugShowCheckedModeBanner: false,
-      title: 'Flutter Demo',
-      theme: ThemeData(
-        primarySwatch: Colors.green,
-      ),
-      
-      home: const AppointmentBooking()
-    );
+        debugShowCheckedModeBanner: false,
```
Comment (jebessaD) [inline afyacare/lib/main.dart:18]: change title to afyacare

**Review by jebessaD — COMMENTED** [review]
all done

File: afyacare/lib/main.dart [hunk 6]
```diff
@@ -14,13 +15,11 @@ class MyApp extends StatelessWidget {
...
+        title: 'Flutter Demo',
```
Comment (jebessaD) [inline afyacare/lib/main.dart:19]: change title to Afya Care

**Review by jebessaD — COMMENTED** [review]
done

**Pull request merged**